packages/okta-react-native/ios/OktaSdkBridge.podspec

```diff
@@ -0,0 +1,22 @@
+
+Pod::Spec.new do |s|
+  s.name         = "OktaSdkBridge"
+  s.version      = "1.0.0"
+  s.summary      = "ROktaSdkBridgeNNewlibrary"
+  s.description  = <<-DESC
+                  RNNewlibrary
+                   DESC
+  s.homepage     = ""
+  s.license      = "MIT"
+  # s.license      = { :type => "MIT", :file => "FILE_LICENSE" }
+  s.author             = { "author" => "author@domain.cn" }
```

Comment:
your email

Reply:
Do we want to keep the author field? I'm deleting it since I don't think its needed